editor.planx.uk/src/pages/Pay/MakePayment.tsx

```diff
@@ -207,3 +265,24 @@ function computePaymentState(govUkPayment: GovUKPayment | null): PaymentState {
   // PaymentStatus.cancelled, PaymentStatus.error, PaymentStatus.failed,
   return PaymentState.Failed;
 }
+
+const parseSessionPreviewData = (sessionPreviewData: unknown) => {
```

Comment:
We could just cast this value to a typed `SessionPreviewData`, or update the type of `PaymentRequest` in `planx-core` but this approach which will make sure we have what's required.